src/LodeRunner/src/Program.cs

```diff
@@ -45,9 +45,13 @@ public sealed partial class App
         private static readonly CancellationTokenSource CancelTokenSource = new ();
 
         /// <summary>
+        /// Prevents a default instance of the <see cref="App"/> class from being created.
         /// Private constructor to prevent instantiation.
         /// </summary>
-        private App() { }
+        /// </summary>
```

Comment:
It looks like there are 2 closing summary tags?

Reply:
good eye